webroot/js/cake-core.js

```diff
@@ -0,0 +1,22 @@
+var elements = document.getElementsByClassName('cake-core-postLink');
```

Comment:
Sry .. I read that over hundred of times now .. but I really don't get it, what you mean. :laughing: Sry.